mne/utils/check.py

```diff
@@ -384,6 +384,11 @@ def _check_pybv_installed(strict=True):
     return _soft_import('pybv', 'exporting to BrainVision', strict=strict)
 
 
+def _check_pymatreader_installed(strict=True):
+    """Aux function."""
+    return _soft_import('pymatreader', 'pymatreader', strict=strict)
```

Comment:
for nice error messages, the second argument to `_soft_import` should be a descriptive noun phrase of what action pymatreader supports. Maybe:

```suggestion
    return _soft_import('pymatreader', 'loading .MAT files', strict=strict)
```
?

Reply:
OK, I guess it should be "loading v7.3 (HDF5) .MAT files" then.